alf/networks/networks_test.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+
+import alf
+
+from alf.networks import TemporalPool
+
+
+class NestworksTest(alf.test.TestCase):
+    def test_temporal_stack_skip(self):
```

Comment:
One minor point is that we can probably change ``test_temporal_stack_max`` to ``test_temporal_pool_max``.
Same for the other tests.